.github/workflows/js-benchmarks.yml

```diff
@@ -36,9 +36,6 @@ jobs:
         if: ${{ matrix.os_name == 'Linux' }}
         shell: bash
         run: |
-          curl -f -o /usr/share/keyrings/llvm-snapshot.gpg.key https://apt.llvm.org/llvm-snapshot.gpg.key
-          echo "deb [signed-by=/usr/share/keyrings/llvm-snapshot.gpg.key] http://apt.llvm.org/noble/ llvm-toolchain-noble-20 main" | sudo tee -a /etc/apt/sources.list.d/llvm.list
-
           sudo apt-get update
           sudo apt-get install -y clang++-20 python3-venv
```

Comment:
I'm just now realizing it, but do we even need to install clang in this workflow? :thinking:

Reply:
Probably not! Not sure why it's in there - if it's required for something else (weird pip requirements?) we should definitely add a comment here.